### PR TITLE
docs: update installation guide nodejs version to 8.6

### PR DESCRIPTION
**Pull request by SukkaW**
https://github.com/hexojs/hexo/issues/3508

- [ ] Read the [theme publishing doc](https://hexo.io/docs/themes#Publishing) or [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).

Update installation guide Node.js version requirements. Should be merged right before the Hexo 4.0.0 is released.

https://github.com/hexojs/hexo/issues/3508

### CODE REVIEW

File: source/zh-cn/docs/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Hexo 是一个快速、简洁且高效的博客框架。Hexo 使用 [Markdown](h
 
 安装 Hexo 相当简单。然而在安装前，您必须检查电脑中是否已安装下列应用程序：
 
-- [Node.js](http://nodejs.org/) (Should be at least nodejs 6.9)
+- [Node.js](http://nodejs.org/) (Should be at least Node.js 8.6)
```
Comment (curbengh) [inline source/zh-cn/docs/index.md]: Maybe a quick translation in the meantime? :wink:

Comment (curbengh) [reply source/zh-cn/docs/index.md]: zh-tw?

Comment (SukkaW) [reply source/zh-cn/docs/index.md]: @curbengh I am not sure the translations for zh-tw is correct, so I leave it untouched.

Comment (curbengh) [reply source/zh-cn/docs/index.md]: I think should be the same (in this case). anyway, I added it.

**Comment by NoahDragon** [comment]
I feel it is okay to merge this info before Hexo@4 released. Please let me know if there are some reasons that we need to hold this now.

**Pull request merged**